port/ztpf/omrosdump_helpers.h

```diff
@@ -0,0 +1,292 @@
+
+#ifndef  TRUE
+#define  TRUE  1
+#endif
+#ifndef	 FALSE
+#define  FALSE 0
+#endif
+
+/**
+ * \internal This macro pushes an address up to the nearest higher
+ *	  paragraph (0x10) boundary.
+ *
+ * \param[in]  a	 Any address/pointer value
+ *
+ * \returns			 The address of the next higher paragraph.
+ */
+#define NEXT_PARA(a) (((((UDATA)(a))+15) >> 4) << 4 )
+
+/**
+ * \internal DBFINULL is a typecast-qualified equivalent of NULL
+ */
+#define DBFINULL	((DBFITEM *)0)
+#define DBTIVADDR	(curr->ijavsad)			/** The starting EVA of the data region.			*/
+#define DBTIOFFS	(curr->ijavsbuf)		/** The starting address in our local copy buffer.	*/
+#define DBTISIZ		(curr->ijavsiz)			/** The size of the memory region					*/
+#define DBTIGAP		(curr->ijavgap)			/** The gap field									*/
+/*
+ * DBITYPE is one byte of bitfields; if you write one zero there, you'll
+ * have turned all its flags off. These bit indicators are symbolized 
+ * for two pointer combinations; the DBITYPE bit addressed by the 
+ * <tt>curr</tt> pointer and that addressed by the <tt>pred</tt> pointer. 
+ * Values starting with 'C' are addressed off the <tt>curr</tt> pointer,
+ * and values starting with 'P' are addressed off the <tt>pred</tt> pointer.
+ *	nSETSTRT is the starting item for a contiguous set
+ *	nSETMIDL is in a set, not the starter, not the ender
+ *	nSETLAST is the last (ender) of a contiguous set
+ *	nSETNONE means this item can be ignored
+ */
+#define DBTITYPE	(curr->ijavdesc+35)	/** A byte of bitfields, as follows:	*/
+#define CSETSTRT	(curr->ijavsset)		/** Starts a set	*/
+#define CSETMIDL	(curr->ijavmset)		/** Middle of a set	*/
+#define CSETLAST	(curr->ijavlset)		/** End of a set	*/
+#define CSETSOLO	(curr->ijavnone)		/** A set of one	*/
+#define PSETSTRT	(pred->ijavsset)		/** Starts a set	*/ 
+#define PSETMIDL	(pred->ijavmset)		/** Middle of a set	*/
+#define PSETLAST	(pred->ijavlset)		/** End of a set	*/
+#define PSETSOLO	(pred->ijavnone)		/** A set of one	*/
+
+#ifdef	 HOLD_PAGE_SIZE
+#undef	 PAGE_SIZE
+#define  PAGE_SIZE	 HOLD_PAGE_SIZE
+#undef	 HOLD_PAGE_SIZE
+#endif
+
+/*
+ *	  Aggregate data type declarations of importance to the ELF core file.
+ *	  Most of these aggregates appear somewhere in the .NOTE section.
+ */
+//avoid: error: conflicting types for greg_t, etc... sys/ucontext.h already has them
+//typedef uint64_t	greg_t;			/* Datatype for a general register			*/
+//typedef SYS_FLOAT	fpreg_t;		/* Datatype for a floating point reg		*/
+					 
+/*					 
+ *	The NT_AUXV segment means something to a UNIX or Linux system. It means
+ *	nothing to z/TPF; we don't turn over control to the system interpreter (we
+ *	don't even HAVE one) like *ix systems do. So instead, we're going to
+ *	fulfill the requirement for an NT_AUXV segment with a fake one of 304 zeros.
+ */
+typedef struct fake_auxv {
+   uint8_t		fake_data[304];		/* Gotta go with 304 zeros here.			*/
+} AUXV;
+
+typedef struct { 
+   uint64_t		hi_word;
+   uint64_t		lo_word;
+} psw_t;
+
+//

```

Comment:
Removed J9ZTPF guard macros and code that should never get built.  We will use the OMRZTPF convention per your suggestion for other segments.